Dockerfile

```diff
@@ -1,56 +1,53 @@
 # syntax=docker/dockerfile:1.4
 
 ####################################################################################################
-## Prepare Workspace
+## Build Packages
 
-FROM node:18-alpine AS workspace
-WORKDIR /workspace
+FROM node:18-alpine AS builder
+WORKDIR /directus
+
+ENV NODE_OPTIONS=--max-old-space-size=8192
 
-COPY pnpm-lock.yaml .
 RUN corepack enable && corepack prepare pnpm@7.30.0 --activate
+
+COPY pnpm-lock.yaml .
 RUN pnpm fetch
 COPY . .
 RUN pnpm install --recursive --offline --frozen-lockfile
 
-####################################################################################################
-## Create Deployment
-
-FROM workspace AS pruned
-WORKDIR /workspace
-ENV NODE_OPTIONS=--max-old-space-size=8192
-
-RUN pnpm --recursive --workspace-concurrency=1 run build
-RUN pnpm --filter directus deploy --prod pruned
-RUN cd pruned \
+RUN : \
```

Comment:
I don't see any reason to keep these layers separate. All of these steps must be performed each time the code changes, so this ought to be a single layer.